### ISSUE TITLE
Support Case Insensitive Security checks

### ISSUE THREAD

**Issue by BenB196** [mainwp/mainwp-child#326]
**Describe the bug**
Today, Security checks appear to be case sensitive; Example: https://github.com/mainwp/mainwp-child/blob/1c176473001fd698e810434463a9e06cff42ebd6/class/class-mainwp-security.php#L454 looks only for `off`, but doesn't account for things like `Off` or `OFF`

**To Reproduce**
1. Setup WordPress
2. Disable PHP logging with non-lowercase value:
```bash
I have no name!@wordpress-78f5777878-p4rwq:/opt/bitnami/wordpress$ cat /opt/bitnami/php/etc/php.ini | grep display_error
; display_errors
display_errors = Off
; separately from display_errors. We strongly recommend you set this to 'off'
I have no name!@wordpress-78f5777878-p4rwq:/opt/bitnami/wordpress$ cat /opt/bitnami/php/etc/php.ini | grep display_startup_errors
; display_startup_errors
display_startup_errors = Off
```
4. Notice on the MainWP Dashboard that it doesn't think PHP error reporting is disabled

**Expected behavior**
I would expect MainWP to perform case insensitive checks for things like these security settings.

**Screenshots**
![image](https://user-images.githubusercontent.com/8277432/182673117-2a6573e5-5fe6-4104-9dc1-2fbcb0641d0e.png)

**Comment by bogdan-mainwp** [comment]
Hi @BenB196,

thanks for reporting this, I will have our dev team check this asap.

**Comment by bogdan-mainwp** [comment]
Fixed in 4.2.6
